stor/swift.py

```diff
@@ -442,7 +442,9 @@ def is_segment_container(self):
         """True if this path is a segment container"""
         container = self.container
         if not self.resource and container:
-            return container.startswith('.segments_') or container.endswith('_segments')
+            return (container.startswith('.segments_')
+                    or container.endswith('_segments')
```

Comment:
Allegedly this or statement is failing our linter.  I'm happy relaxing but up to you.